docs/standard/commandline/define-commands.md

```diff
@@ -146,6 +148,15 @@ Options:
   -?, -h, --help          Show help and usage information
 ```
 
+
+## Set argument arity
+
+You can explicitly set argument [arity](syntax.md#argument-arity) by using the `Arity` property, but in most cases that is not necessary. `System.CommandLine` automatically determines the argument arity based on the argument type:
+
+* `Boolean` arguments - `ZeroOrOne`
```

Comment:
Are these enum fields or ? If so, maybe put this info in a table with a column header.

Reply:
They are static members of the `ArgumentArity` type.

Reply:
Good idea. It's not an enum but it works like one.